modules/Microsoft.Network/virtualNetworks/deploy.bicep

```diff
@@ -19,6 +19,23 @@ param ddosProtectionPlanId string = ''
 @description('Optional. Virtual Network Peerings configurations.')
 param virtualNetworkPeerings array = []
 
+@description('Optional. Indicates if VM protection is enabled for all the subnets in the virtual network.')
+param enableVmProtection bool = false
+
+@description('Optional. Indicates if encryption is enabled on virtual network and if VM without encryption is allowed in encrypted VNet. Requires the EnableVNetEncryption feature to be registered for the subscription and a supported region to use this property.')
+param vnetEncryption bool = false
```

Comment:
Same comment as for the VM protection

Reply:
But I guess it's a bit tricky if it must be enabled on the subscription

Reply:
Yes not an out of the box feature